packages/ui/atomic/create-atomic/src/fetch-page.ts

```diff
@@ -7,20 +7,32 @@ import {
 /**
  * @coveo/platform-client's IManifestResponse with simplified configuration
  */
-export interface IManifest extends Omit<IManifestResponse, 'config'> {
-  config: Pick<ISearchInterfaceConfigurationResponse, 'title'>;
+export interface IManifest extends Omit<IManifestResponse<never>, 'config'> {
+  config: Pick<ISearchInterfaceConfigurationResponse, 'name'>;
 }
 
 export async function fetchPageManifest(
   client: PlatformClient,
-  pageId: string
+  pageId: string,
+  type: 'next-gen' | 'legacy' | 'unknown'
 ) {
-  const manifest = await client.searchInterfaces.manifest(pageId, {
-    pagePlaceholders: {
-      results: '--results--',
-    },
-  });
-  return replaceResultsPlaceholder(manifest);
+  let manifestGetters = [];
```

Comment:
Do we really still need the support for `legacy` ? 🤔 I don't see where we would be using or needing it in the CLI.

Could we just delete all the code related to downloading legacy page (function `getLegacyManifest` in this file) ?

Reply:
Yes imho. Removing this would be a breaking change for people using the legacy search pages still.
It's important to remember that, from the user perspective, there's no distinction between legacy and next Gen search pages in the UI or anywhere else.